modules/fundamental/src/source_document/model/mod.rs

```diff
@@ -5,9 +5,12 @@ use trustify_common::id::{Id, IdError};
 use trustify_entity::source_document;
 use trustify_module_storage::service::StorageKey;
 use utoipa::ToSchema;
+use uuid::Uuid;
 
 #[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
 pub struct SourceDocument {
+    #[serde(skip)]
+    pub id: Uuid,
```

Comment:
Do we still need this now?

Reply:
The `skip` attribute made me wonder. It's just an internal thing, not visible on the outside. So why have it? Doesn't feel right.